components/DiffMap.vue

```diff
@@ -1,10 +1,11 @@
 <script lang="ts">
 import type { PropType } from 'vue'
 import { shallowRef } from 'vue'
-import type { Geometry } from 'geojson'
+import type { Geometry } from '@turf/helpers'
```

Comment:
I prefer keeing the type from geojson, the solution is probably to cast from/to turf.

Reply:
I think the problem is we are talking about the same type: Geometry but `booleanEqual` turf function expect his own ...
So we have a type error: 'Geometry is not assignable to type Geometry'...

Reply:
```
import type { Geometry } from 'geojson'
import type { Geometry as TurfGeometry } from '@turf/helpers'
```
Is not a solution ?

Reply:
yes it is but a bit too much no ? why not using turf type directly as it is provided ?

Reply:
alright